libr/core/disasm.c

```diff
@@ -1598,6 +1601,9 @@ static void ds_print_show_cursor(RDisasmState *ds) {
 			res[i] = '0' + (diff % 10);
 		}
 	}
+	if (has_hint) {
+		strcpy (res, res + 3);
```

Comment:
I think this involves moving data between overlapping strings, right? If so, man page says that `strcpy`' strings may not overlap.

Reply:
Indeed. Should be using memmove. Do u think we should provide r_str_cpy as a safe alternative?

Reply:
There is already:
```C
#define r_str_cpy(x,y) memmove(x,y,strlen(y)+1);
```

Reply:
i see some missing parenthesis in this macro. ill fix that too